games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/freegish/freegish.git#f53d915381c2dd9889dbebdff0541b04c6065067"
+PATCHES="freegish-1.0.patchset"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	freegish = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libogg
+	lib:openal
+	lib:libpng
+	lib:libsdl
+	lib:libvorbis
+	lib:zlib
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libogg
+	devel:libopenal
+	devel:libpng
+	devel:libsdl
+	devel:libvorbis
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc
+"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DDATAPATH='"$appsDir"'
+	make
+}
+
+INSTALL()
+{
+	cd ..
+	mkdir -p $appsDir/freegish
+	cp -r  freegish-1.0 $appsDir/freegish
+	addAppDeskbarSymlink $appsDir/freegish/ "FreeGish"
```

Comment:
Uh...if I am not wrong the executable made wasn't called FreeGish, its called freegish...so...I will use that capitalization in my change.